pkg/acsengine/json.go

```diff
@@ -41,6 +41,24 @@ func PrettyPrintJSON(content string) (string, error) {
 	return string(prettyprint), nil
 }
 
+// BuildAzureParametersFile will add the correct schema and contentversion information
+func BuildAzureParametersFile(content string) (string, error) {
+	var parametersMap map[string]interface{}
+	if err := json.Unmarshal([]byte(content), &parametersMap); err != nil {
+		return "", err
+	}
+	parametersAll := map[string]interface{}{}
+	parametersAll["$schema"] = "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#"
+	parametersAll["contentVersion"] = "1.0.0.0"
+	parametersAll["parameters"] = parametersMap
```

Comment:
so, this be something like parameters : { parameters :{ ...}}

Are we sure, this is what we want?